test/tools/cli/IonProcess.java

```diff
@@ -392,15 +389,22 @@ private static void embeddedEventToIon(IonReader ionReader,
                         depth--;
                         tempWriter.stepOut();
                     } else if (event.getEventType() == EventType.SYMBOL_TABLE) {
-                        //TODO symbol table
+                        ionWriterForOutput.finish();
```

Comment:
This should be `close()`, which finishes the writer and frees any resources it consumes.

Reply:
Changed